README.md

```diff
@@ -20,6 +20,44 @@ If using a recent version that depends on the Git CLI, install Git with your dis
 
 If using an older release with cgo libgit or native golang Git, the binary will work standalone.
 
+Calculating Tags
+================
+
+The `autotag` utility will use the current state of the git repository to determine what the next tag should be (when following SemVer 2.0).
+Tags created by `autotag` have the following format: `vMajor.Minor.Patch` (e.g., `v1.2.3`).
+
+By default, `autotag` only scans the `master` branch for changes. The utility first looks to find the most-recent reachable tag, only
```

Comment:
I was referring to this configuration option:

* https://github.com/pantheon-systems/autotag/blob/e58f92fbca9b7f1b10862806bd6b2a6966c7988f/autotag/main.go#L17

It's possible I misunderstood its purpose.